### DISCUSSION TITLE
Feature Request - Disable Recursive Execution Method?

### DISCUSSION THREAD

**Discussion by poisenbery** [comfyanonymous/ComfyUI#2525] [Ideas]
I have a node tree that is >250 nodes. It takes around 90 seconds to generate the image. However, when I start to add additional nodes, ComfyUI will spend ~15 minutes validating the input data before generating anything.

Image of workflow: 
![image](https://github.com/comfyanonymous/ComfyUI/assets/78466408/ee8c3f1a-aef7-4501-b77c-910ea3e07b2e)

Everything is necessary.

I think I found the upper limit of ComfyUI with this workflow, but this is only ~10% of what I was hoping to accomplish.

The node tree actually runs faster when it starts from the very beginning. 

is there a way to disable the recursive execute method and just have it run from start to finish every time?

LtDrData made a post about the [execution method](https://github.com/comfyanonymous/ComfyUI/discussions/496) and I think I managed to finally approach the level where the execution method is causing more issues than good.

**Comment by ltdrdata** [comment]
The best suggested solution so far is https://github.com/comfyanonymous/ComfyUI/pull/931. However, the current state is broken as the latest commits need to be integrated. The developer of this PR seems to be busy at the moment, leading to a slow progress.

**Reply by poisenbery** [reply]
Funny you mention, because I actually tried pulling that yesterday with no luck
